### PR TITLE
Added no-extra-semicolons rule to enforce no extra semicolons

### PR DESCRIPTION
**Pull request by benbcai**
### Summary
Added the [no-extra-semicolons](https://stylelint.io/user-guide/rules/no-extra-semicolons) rule to enforce no extra semicolons.

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: stylelint.config.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = {
   ],
   rules: {
     'max-nesting-depth': 3,
+    'no-extra-semicolons': true,
```
Comment (StephenEsser) [inline stylelint.config.js]: This will be non-passive, can we update it to a warning until the next major version bump?

Comment (benbcai) [reply stylelint.config.js]: Good point! https://github.com/cerner/stylelint-config-terra/pull/31/commits/f543f74e335ea2681ce7769f8f3ebf164e05c0ba

**Review by StephenEsser — APPROVED** [review]

**Review by bjankord — APPROVED** [review]

File: stylelint.config.js (hunk 1 above)
Comment (emilyrohrbough) [reply stylelint.config.js]: Did we log an issue to fail this on the next MVB?

Comment (bjankord) [reply stylelint.config.js]: https://github.com/cerner/stylelint-config-terra/issues/32

**Pull request merged**